playbooks/devel.yml

```diff
@@ -1,21 +1,28 @@
 ---
-- hosts: devel
+- hosts: all
   become: true
   vars:
     katello_repositories_version: nightly
-    foreman_installer_scenario: katello-devel
-    foreman_installer_options_internal_use_only:
-      - "--disable-system-checks"
-      - "--katello-devel-github-username {{ katello_devel_github_username }}"
-      - "--katello-devel-enable-ostree=true"
+    katello_repositories_environment: staging
```

Comment:
New default, right? Do we expect people to set this to 'production' after this change?

Reply:
Only cause right now nightly is perpetually broken.